controllers/cloud.redhat.com/providers/sidecar/provider.go

```diff
@@ -1,10 +1,33 @@
 package sidecar
 
 import (
+	crd "github.com/RedHatInsights/clowder/apis/cloud.redhat.com/v1alpha1"
+	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/clowderconfig"
 	"github.com/RedHatInsights/clowder/controllers/cloud.redhat.com/providers"
 )
 
-var DefaultImageSideCarTokenRefresher = "quay.io/observatorium/token-refresher:master-2023-09-20-f5e3403" // nolint:gosec
+var DefaultImageSideCarTokenRefresher = "quay.io/observatorium/token-refresher:master-2022-10-21-a99ce82"                               // nolint:gosec
+var DefaultImageSideCarOtelCollector = "ghcr.io/os-observability/redhat-opentelemetry-collector/redhat-opentelemetry-collector:0.107.0" // nolint:gosec
+
+func GetFeatureFlagsUnleashImage(env *crd.ClowdEnvironment) string {
+	if env.Spec.Providers.Sidecars.TokenRefresher.Image.TokenRefresherImg != "" {
```

Comment:
Given my suggestion above,
this then would look like
```if env.Spec.Providers.Sidecars.TokenRefresher.Image```
Which reads better and is more succinct